client/src/containers/CampaignAdmin/CampaignAdminList.js

```diff
@@ -66,13 +66,28 @@ class CampaignAdminList extends SilverStripeComponent {
     this.props.breadcrumbsActions.setBreadcrumbs(breadcrumbs);
   }
 
+  getSelectedItem() {
+    const itemId = this.props.campaign.changeSetItemId;
+    const items = this.getItems() || [];
+    let selected = null;
+
+    selected = items.find(item => itemId === item.ID);
```

Comment:
I think that would be unnecessary. What would be an edge case for not having the check?